### DISCUSSION TITLE
How enable Double Commander internal terminal ?

### DISCUSSION THREAD

**Discussion by Augusto7743** [doublecmd/doublecmd#1572] [General]
Hello. Thanks for reading my question.
Here using current 1.1.12.
When trying start terminal in an path folder is opened the Ubuntu terminal window.
However how configure to use Double Commander's terminal ?
Using the Double Commander QT version and configured terminal to
x-terminal-emulator

Have an nice week.

**Comment by alexx2000** [comment]
Hi, see [Show terminal window](https://doublecmd.github.io/doc/en/configuration.html#ConfigLayout) option.

**Comment by Augusto7743** [comment]
Thanks for your reply.
are being used "x-terminal-emulator" and even so not works.

**Reply by alexx2000** [reply]
What do you mean? Internal terminal emulator does not depend on the `x-terminal-emulator`. It displays in main DC window at the bottom.

**Comment by Augusto7743** [comment]
Thanks for your reply.
Closing.
